clients/ui/manifests/base/kustomization.yaml

```diff
@@ -2,15 +2,18 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - model-registry-bff-role.yaml
-  - model-registry-bff-service.yaml
-  - model-registry-bff-deployment.yaml
-  - model-registry-ui-service.yaml
-  - model-registry-ui-deployment.yaml
-  - model-registry-service-account.yaml
+- model-registry-bff-role.yaml
+- model-registry-bff-service.yaml
+- model-registry-bff-deployment.yaml
+- model-registry-ui-service.yaml
+- model-registry-ui-deployment.yaml
+- model-registry-service-account.yaml
 
 images:
-  - name: model-registry-ui-image
-    newName: kubeflow/model-registry-ui:latest
-  - name: model-registry-bff-image
-    newName: kubeflow/model-registry-bff:latest
+- name: model-registry-bff-image
+  newName: kubeflow/model-registry-bff
+  newTag: latest
+- name: model-registry-ui-image
+  newName: kubeflow/model-registry-ui
+  newTag: test-eder
+namespace: latest
```

Comment:
Could you give some context for these changes?

Reply:
Yeah, that's a typo, already fixed, thanks @Griffin-Sullivan